Packs/Akamai_SIEM/Integrations/Akamai_SIEM/Akamai_SIEM.py

```diff
@@ -122,14 +124,16 @@ def get_events_with_offset(
             resp_type='text',
         )
         demisto.info("Finished executing request to Akamai, processing")
-        events: list[dict] = []
-        for event in raw_response.split('\n'):
-            try:
-                events.append(json.loads(event))
-            except Exception as e:
-                if event:  # The last element might be an empty dict.
-                    demisto.error(f"Could not decode the {event=}, reason: {e}")
-        offset = events.pop().get("offset")
+        events: list[str] = raw_response.split('\n')
+        offset = None
+        try:
+            if events and events[-1] == "":
+                events.pop()
```

Comment:
whats this do?

Reply:
Just a protection mechanism, when we get empty response or something like that.